.github/scripts/build.functions.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Checks if we should build the OSS docker image.
+# Returns "yes" if we should build it or "no" if we shouldn't.
+function should_build_oss() {
+
+  local release_type=$1
+  if [[ $release_type == "ALL" || $release_type == "OSS" ]]; then
+    echo "yes"
+    return
+  fi
+
+  echo "no"
```

Comment:
Nit - why the early return? Why not just if / else?

Reply:
Just a matter of the style, there were more conditions and I wanted to avoid nesting. See e.g.: https://stackoverflow.com/a/9267684

Reply:
Fixed: https://github.com/hazelcast/hazelcast-docker/pull/750/commits/ad727708e672a07b796b55cff4cccd90b6441950